[WIP] Update Contributing Doc (#213)
tektoncd/website: CONTRIBUTING.md

To get started developing, see our [DEVELOPMENT.md](./DEVELOPMENT.md) and [DOC-CON-MAIN.md](./content/en/doc-con-main.md).

The [OWNERS](OWNERS) of this repo are:
- [members of the Tekton governing board](https://github.com/tektoncd/community/blob/master/governance.md)

-  regular contributors who were elected by the Tekton governing board. 

For a contributor to be noimated as an owner for the tekton website repo:

- create 5 pull requests for the tekton website
- regulary attend the docs/website working group

> 
> attend the as well as folks from the Linux Foundation who contribute to this project on a regular basis. 

Reviewer: ```suggestion
- member of the Linux Foundation who contributes to the project on a regular basis. 
```
